src/plugins/wb-postback/wb-postback.js

```diff
@@ -42,6 +42,16 @@ var $document = wb.doc,
 				// Prevent regular form submit
 				e.preventDefault();
 
+				//Check if the form use the validation plugin
+				if ( elm.parentElement && elm.parentElement.classList.contains( "wb-frmvld" ) ) {
```

Comment:
No need to check for `elm.parentElement`, this will be only false for the HTML element

Reply:
Fixed. Thanks